web-src/src/mystyles.scss

```diff
@@ -3,6 +3,29 @@
 @import 'bulma/bulma.sass';
 @import 'bulma-switch';
 
+.is-no-basis {
+  flex-basis: 0;
+}
+
+/* Fix bulma issue for a group under a navbar-brand container (contrary to documentation) */
+.navbar.is-white .navbar-brand > .navbar-item *:focus,  .navbar.is-white .navbar-brand >  .navbar-item *:hover, .navbar.is-white .navbar-brand > .navbar-item *.is-active {
+  color: #0a0a0a;
```

Comment:
Can you please use the variables for colours?
Thanks.

Reply:
Which one ?

This is a copy and paste from bulma's code to fix a bug in their declaration. It should work with a ["container with anything you want"](https://bulma.io/documentation/components/navbar/#navbar-item) but it doesn't in reality. 

I've fixed the CSS rule to support a container correctly. The content of the rule is a copy and paste from bulma's CSS. 

I haven't tried using $white or $grey because I don't know if this variable that's defined in bulma is available in this file. If it is, which one should I use ?

Reply:
I would use `$white` for the white (#ffffff) and `$black` for the kind of "dark grey" (#0a0a0a).
The Bulma variables can be used at this place, because they're imported with the statement `@import 'bulma/bulma.sass';`

I also made a slight change in the use of color variables in the main branch, because they were referenced incorrectly: `#{white}` instead of `$white` and `#{$danger}` instead of `$danger`

The Bulma colors are available [here](https://bulma.io/documentation/overview/colors/).

Reply:
Done